### PR TITLE
[ADD] stock_split_quant_by_lot: new module

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

**Comment by agaldona** [comment]
@avanzosc/developers review when possible please

File: stock_split_quant_by_lot/wizard/wizard_load_quant_lot_lines.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+    def reasign_quants(self, move, inventory, before_quants):
+        inventory_quants = inventory.mapped('move_ids.quant_ids')
+        quants = self.env['stock.quant']
+        quants += before_quants
+        quants += inventory_quants.filtered(
+            lambda x: x.location_id.usage == 'internal')
+        for quant in quants:
+            move.quant_ids = [(4, quant.id)]
+        for quant in move.quant_ids:
```
Comment (alfredoavanzosc) [inline stock_split_quant_by_lot/wizard/wizard_load_quant_lot_lines.py]: ¿Valdría esto para quitarte la if?
for quant in move.mapped('quant_ids').filtered(lambda l: l.location_usage != 'internal' and l.id in inventory_quants):

**Review by Daniel-CA — DISMISSED** [review]

**Review by oihane — DISMISSED** [review]
LGTM

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

**Review by Daniel-CA — APPROVED** [review]

**Review by oihane — APPROVED** [review]

**Pull request merged**